### DISCUSSION TITLE
Library Back Button - Folder Navigation

### DISCUSSION THREAD

**Discussion by markblinkhorn** [rern/rAudio#1849] [Fixed Issues]
When moving back to a previous folder level in the library - either by using the back button or by clicking on the path list at the top - the cursor position is reset back to the top of the list. This isn't a problem when there is only a limited number of items in the list but does become an issue at the root level of my library which contains hundreds of items (folders). Previous versions of rAudio retained the previous cursor position but I'm not certain which version changed the behaviour.

I hadn't realised how tricky it is to describe this issue until I tried! I hope you can make sense of my ramblings.

**Comment by rern** [comment]
Please try the UPDATE.
(Long-press `Update` > `Ok` to branch 'UPDATE')

**Comment by markblinkhorn** [comment]
UPDATE restores previous behaviour. Thanks.
